packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -877,8 +872,6 @@ export const useArbTokenBridge = (
 
   const updateTokenData = useCallback(
     async (l1Address: string) => {
-      updateUSDCBalances(l1Address)
```

Comment:
This didn't work at all because l1Address can be a different token address and then it does nothing.